### DISCUSSION TITLE
How to obtain volume material output of the mesh from Seissol?

### DISCUSSION THREAD

**Discussion by jwjeremy** [SeisSol/SeisSol#1063] [Q&A]
I am asking how to configure seissol parameter file to write the volumetric output of the material properties of the mesh? I would like to visualize the 3D velocity structure as shown in this beautiful mesh plot in Taufiqurrahman, et al. "Broadband dynamic rupture modeling with fractal fault roughness, frictional heterogeneity, viscoelasticity and topography: The 2016 Mw 6.2 Amatrice, Italy earthquake." Geophysical Research Letters 49.22 (2022): e2022GL098872.

Thanks! 

![image](https://github.com/SeisSol/SeisSol/assets/44199955/d040ba90-076b-4f33-a761-50648d2c34c6)

**Comment by fabian-kutschera** [comment]
Hi @jwjeremy, I don't know which exact script Taufiq used, but check out this link: https://github.com/SeisSol/Meshing/tree/master/evaluate_material

**Reply by jwjeremy** [reply]
Awesome! Thanks so much! I will check it out!

**Comment by jwjeremy** [comment]
@sebwolf-de I got a fatal error of missing utils/args.h when compiling the evaluate_material in LMU exception server. Do you have any suggestions? Many thanks! 
![image](https://github.com/SeisSol/SeisSol/assets/44199955/cb6bc5e2-b231-4f0f-a75e-7c1e54e9cb31)

**Reply by sebwolf-de** [reply]
Try `git submodule update --init`
